### PR TITLE
Expose POST /tenants (create org)

### PR DESCRIPTION
**Pull request by PiotrPrzybylak**
Issues: MEN-1649

Changelog: None
Signed-off-by: Piotr Przybylak <piotr.przybylak@gmail.com>

### CODE REVIEW

File: tenantadm.nginx.conf [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
+location = /api/management/v1/tenantadm{
     auth_request /userauth;
     auth_request_set $requestid $upstream_http_x_men_requestid;
 
     proxy_pass http://mender-tenantadm:8080;
     proxy_set_header X-MEN-RequestID $requestid;
 }
+
+location = /api/management/v1/tenantadm/tenants{
+    proxy_pass http://mender-tenantadm:8080;
```
Comment (maciejmrowiec) [inline tenantadm.nginx.conf]: should we add check for method just in case for the future?

```
 if ($request_method = POST ) {
....
}
```

If we would have for some reason GET on /tenants route that would get opened it could be bad

File: tenantadm.nginx.conf [hunk 2]
```diff
@@ -1,7 +1,11 @@
-location = /api/management/v1/tenantadm/user/tenant{
+location = /api/management/v1/tenantadm{
     auth_request /userauth;
     auth_request_set $requestid $upstream_http_x_men_requestid;
 
     proxy_pass http://mender-tenantadm:8080;
     proxy_set_header X-MEN-RequestID $requestid;
```
Comment (mchalski) [inline tenantadm.nginx.conf]: this is how we do it, but maybe it makes sense to check if we can get rid of `auth_request_set` extra var.

File: tenantadm.nginx.conf [hunk 3]
```diff
@@ -1,7 +1,16 @@
...
 }
+
+location = /api/management/v1/tenantadm/tenants{
+
+    if ($request_method != POST) {
+        return 403;
```
Comment (maciejmrowiec) [inline tenantadm.nginx.conf]: 405 Method Not Allowed

File: tenantadm.nginx.conf [hunk 4]
```diff
@@ -1,7 +1,16 @@
-location = /api/management/v1/tenantadm/user/tenant{
+location = /api/management/v1/tenantadm{
```
Comment (mchalski) [inline tenantadm.nginx.conf]: `=` means 'exact match', so it's no good for matching 'any endpoint on tenantadm' (which I assume was the intention). imo this location will match only requests to `/api/management/v1/tenantadm`, which will never happen, this is just the root api url.

a 'prefix location' would be more appropriate (no modifier). it would match all requests starting with `/api/management/v1/tenantadm` - with the exception of the `/tenants` endpoint. check out:
https://www.digitalocean.com/community/tutorials/understanding-nginx-server-and-location-block-selection-algorithms (esp `How Nginx Chooses Which Location...`).

in short, the rule matching would then go like this:
* is there an exact match? (`=`) if yes, use the location immediately (handles the `/tenants` ep)
* find the longest matching prefix location, save for later (would save `location  /api/management/v1/tenantadm`)
* try match regex locations (none in our case)
* go back to the longest prefix location (this one), use that

Comment (PiotrPrzybylak) [reply tenantadm.nginx.conf]: Fixed
